docs2/site/docs/migrations/migration4.md

```diff
@@ -540,6 +541,8 @@ default continues to be `ResolverType.Resolver`.
 * `IResolveFieldContext.Fragments` was removed; use `IResolveFieldContext.Document.Fragments` instead
 * `ExecutionContext.Fragments` was removed; use `ExecutionContext.Document.Fragments` instead
 * `AbstractGraphTypeExtensions.GetTypeOf` was removed; use `AbstractGraphTypeExtensions.GetObjectType` instead
+* `TypeConfig.FieldFor(string, IServiceProvider)` and `TypeConfig.SubscriptionFieldFor(string, IServiceProvider)` 
+  methods were merged into single `TypeConfig.FieldFor(string)` method and just return the required configuration without its initialization
```

Comment:
This would more appropriately say that the methods were removed, if they are not going to be protected.

Reply:
This method remained public. It is needed to set the configuration, not internal things.

Reply:
Oh ok 👍